### PR TITLE
Instructions for removing the Edit on Github link

### PR DESCRIPTION
**Pull request by chloepoulterdesign**
I added a line to help people take out the relevant part of gatsby-config.js to take out the "Edit on Github" link from their sites.

### CODE REVIEW

**Comment by vpicone** [comment]
We might want to make sure they don’t have any other options defined. Something like “delete the repository option. If after deleting, your options object is empty, delete that as well.”

**Comment by chloepoulterdesign** [comment]
Thanks for the feedback @vpicone .
I've updated and committed again in light of your feedback.

File: packages/example/src/pages/guides/configuration.mdx [hunk 1]
```diff
@@ -143,6 +143,8 @@ To do so, provide your own [resolvers object](https://www.gatsbyjs.org/packages/
 
 To add a link to the bottom of each page that points to the current page source in GitHub, provide a `repository` object to `siteMetadata` in your `gatsby-config.js` file. You can provide a `baseUrl`, and if needed, the `subDirectory` and `branch` where your site source lives.
 
+To remove the link, remove the section that begins `options` through to its' closing brackets. If after deleting, your options object is empty, delete that as well.
```
Comment (vpicone) [inline packages/example/src/pages/guides/configuration.mdx:146]:
```suggestion
To disable the `Edit in Github` link, remove the `repository` option. If the options object is now empty, delete it all together.
```

Comment (chloepoulterdesign) [reply packages/example/src/pages/guides/configuration.mdx:146]: I'm happy with this - it's just that to me an "option" doesn't mean anything - I wouldn't know that that means I need to remove several lines of code 🤷‍♀️.
I'm probably not your usual audience though 🙂.

**Pull request merged**